src/server.ts

```diff
@@ -298,8 +298,9 @@ const defaultMapErrorToJSONRPCErrorResponse = (
 ): JSONRPCErrorResponse => {
   return createJSONRPCErrorResponse(
     id,
-    DefaultErrorCode,
-    (error && error.message) || "An unexpected error occurred"
+    error?.code || DefaultErrorCode,
+    error?.message || "An unexpected error occurred",
+    error?.data
```

Comment:
We don't know if this `error` is `JSONRPCErrorException`. Can you please add `code` and `data` only when it is from `JSONRPCErrorException`?

Reply:
I am assuming you are concerned that `code` and `data` might mean something else in a different context so I have made the same change to `message` for the same reason.

Reply:
Good point. Thanks!

Reply:
> I have made the same change to message for the same reason since error can be of any type.

Actually I'll revert this change in v1.4.1. I just realized this will be a breaking change.